cloudinit/sources/DataSourceOVF.py

```diff
@@ -318,6 +324,20 @@ def get_public_ssh_keys(self):
     def get_config_obj(self):
         return self.cfg
 
+    def _raise_error_status(self, prefix, error, event, config_file):
```

Comment:
Why lift this into the class?  You could just add a parameter to the _raise_error_status to accept the flag to indicate if you need to report status.

Reply:
Thanks Ryan, I just changed the codes and committed, please take another look. Thanks